scripts/database/create_tables.sql

```diff
@@ -98,3 +98,24 @@ BEGIN
     );
 END
 
+IF NOT EXISTS
+(
+    SELECT *
+    FROM INFORMATION_SCHEMA.TABLES
+    WHERE TABLE_SCHEMA = 'dbo'
+    AND TABLE_NAME = 'ORGANISATION_LKP'
+)
+BEGIN
+    CREATE TABLE ORGANISATION_LKP
+    (
+      ORGANISATION_ID NVARCHAR (50) NOT NULL,
+        CONSTRAINT PK_ORGANISATION_ID
+          PRIMARY KEY (ORGANISATION_ID),
+      SCREENING_NAME                  VARCHAR(50) NULL,
+      ORGANISATION_CODE               VARCHAR(50) NULL,
+      ORGANISATION_NAME               VARCHAR(50) NULL,
+      ORGANISATION_TYPE               VARCHAR(50) NULL,
+      IS_ACTIVE                       VARCHAR(50) NULL,
+    );
+END
+
```

Comment:
Could you also add a few insert statements so we have some records in this table, please